### PR TITLE
Feature/56489 - Pagination

### PR DESCRIPTION
**Pull request by jszczech**

### CODE REVIEW

File: src/molecules/pagination/Pagination.js [hunk 1]
```diff
@@ -1,14 +1,23 @@
 // @vue/component
 export default {
   props: {
+    /*
+     * Current page
+     */
     page: {
```
Comment (szafran89) [inline src/molecules/pagination/Pagination.js]: change to `currentPage`

File: src/molecules/pagination/Pagination.js [hunk 2]
```diff
@@ -1,14 +1,23 @@
 // @vue/component
 export default {
   props: {
+    /*
+     * Current page
+     */
```
Comment (szafran89) [inline src/molecules/pagination/Pagination.js]: Wrong comment, it's not visible in vue info tab

File: src/molecules/pagination/Pagination.js [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
     page: {
       type: Number,
       required: true
     },
+    /*
+     * Total size
+     */
```
Comment (szafran89) [inline src/molecules/pagination/Pagination.js]: Wrong comment, it's not visible in vue info tab

File: src/molecules/pagination/Pagination.js [hunk 4]
```diff
@@ -1,14 +1,23 @@
...
     totalSize: {
       type: Number,
       required: true
     },
+    /*
```
Comment (szafran89) [inline src/molecules/pagination/Pagination.js]: Wrong comment, it's not visible in vue info tab

File: src/molecules/pagination/Pagination.js [hunk 5]
```diff
@@ -1,14 +1,23 @@
...
+     * Limit on page
+     */
     limit: {
```
Comment (szafran89) [inline src/molecules/pagination/Pagination.js]: Why you added this prop. In my opinion, we just need a total number of pages and current page props

File: src/molecules/pagination/Pagination.js [hunk 6]
```diff
@@ -1,14 +1,23 @@
...
+     * Current page
+     */
     page: {
       type: Number,
       required: true
     },
+    /*
+     * Total size
+     */
     totalSize: {
```
Comment (szafran89) [inline src/molecules/pagination/Pagination.js]: we don't need the total number of products and limit per page

File: CHANGELOG.md [hunk 7]
```diff
@@ -51,7 +52,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Remove 'selected' story in Selected coomponents
 - Refactor styles in Select component
 - Update modal component js and styles to handle sidebar case
-- Update attributes passed to Select component ($attrs)
```
[the remark this answers is not in the document]
Comment (jszczech) [reply CHANGELOG.md]: 😢 😞 🙍

[the remark this answers is not in the document]
Comment (jszczech) [reply CHANGELOG.md]: 😢 😞  Honestly, I don't know 🙍

**Pull request merged**